### DISCUSSION TITLE
Is it true that you will fix the 2/2 booster vaccination validity issue?

### DISCUSSION THREAD

**Discussion by Ein-Tim** [Digitaler-Impfnachweis/certification-apis#208] [Q&A] [answered]
This article: https://www.stuttgarter-zeitung.de/inhalt.booster-nach-johnson-ungueltig-mhsd.9c9a27cc-2866-4d1f-b93d-f5a622ab47b0.html states:
> Eine Servicemitarbeiterin der CovPass-App bestätigte uns am Telefon, dass die Entwickler bereits an einer Lösung arbeiten, sodass die App zwischen Zweitimpfung und Auffrischungsimpfung unterscheiden kann.

Is this true? If yes, how will you fix this?

**Comment by Ein-Tim** [comment, marked as answer]
Okay, the EU decided to change how booster vaccination certificates are issued if the booster is a vaccination on top of a 1/1 basic vaccination. See https://ec.europa.eu/commission/presscorner/detail/en/ip_21_6837:

"Boosters will be recorded as follows:

- 3/3 for a booster dose following a primary 2-dose vaccination series.
- 2/1 for a booster dose following a single-dose vaccination or a one dose of a 2-dose vaccine administered to a recovered person."

**Comment by JDoodsCGM** [comment]
Dear sender, 

due to a ransomware attack and necessary security measures, CGM is temporarily not reachable in some regions via the usual eMail addresses and landline numbers. 
In these cases, please feel free to use SMS and mobile telephony of your individual contact persons, as far as they are known to you or inform yourself about further contact possibilities via the respective company or product page. 

Thank you for your understanding and best regards,
Your CGM Team

----------------------

Sehr geehrte Absenderin, 
sehr geehrter Absender,  

aufgrund einer Ransomware-Attacke und notwendiger Sicherheitsmaßnahmen, ist CGM in einigen Regionen vorübergehend nicht über die gewohnten eMail-Adressen und Festnetznummern erreichbar. 
Bitte nutzen Sie gerne in diesem Fall SMS und Mobilfunk-Telefonie Ihrer individuellen Ansprechpartner, soweit diese ihnen bekannt sind oder informieren Sie sich über die jeweilige Firmen- bzw. Produktseite über weitere Kontaktmöglichkeiten. 

Vielen Dank für Ihr Verständnis und freundliche Grüße, 
Ihr CGM-Team
